vignettes/imcRtools.Rmd

```diff
@@ -832,6 +837,28 @@ For each patch larger than 2 cells, the spatial area can be computed using the
 patchSize(pancreasSCE)
 ```
 
+## Minimal distances
+
+To calculate the minimal distance for each cell to a given cell type or class of
+cells of interest, the function `minDistToCells` is available. Cells of interest
+are defined via the `x_cells` parameter as `logical` and distances to for all
+cells to those cells will be reported in a new column in the `colData` of the
+`SingleCellExperiment`.
+
+If the cells of interest form patches the positive distances reflect the
```

Comment:
I find it a bit hard to understand how "patches" are defined here. Maybe better to just talk about pos/neg distances and only mention that this is preferable if patches were previously detected.